pybeerxml/recipe.py

```diff
@@ -92,6 +92,15 @@ def fg(self):
 
         return _fg
 
+    @property
+    def color(self):
+        # Formula source: http://brewwiki.com/index.php/Estimating_Color
+        mcu = 0
+        for f in self.fermentables:
+            mcu += f.amount * f.color * 8.3453 / self.batch_size
```

Comment:
The default values for `amount` and `color` are `None`. ([see here](https://github.com/hotzenklotz/pybeerxml/blob/master/pybeerxml/fermentable.py#L13)) Perhaps we should be a bit more defensive here and check that both properties are not `None`. What do you think?

Reply:
Agreed! I'll update the code.